src/library/Uno.Material/Styles/Controls/v2/Flyout.xaml

```diff
@@ -245,6 +245,7 @@
 						<TextBlock x:Name="TextBlock"
 						           Grid.Column="1"
 						           Margin="12,0,0,0"
+								   TextWrapping="NoWrap"
```

Comment:
If everything works on Windows without this change, then this change masks an Uno bug and is only a workaround not a fix for the underlying issue.

Reply:
well, it actually doesn't work on GTK, all the platforms works without this change :/

Reply:
Then this is only a workaround. In case we went we such a workaround, make sure to have an issue open in unoplatform/uno repo and link the issue in a comment here.